ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -62,6 +68,18 @@ export const CronWorkflowDetails = ({match, location, history}: RouteComponentPr
 
     useEffect(() => setEdited(true), [cronWorkflow]);
 
+    useEffect(() => {
+        const getWorkflowsList = async () => {
+            const workflowList = await services.workflows.list(namespace, null, [`${models.labels.cronWorkflow}=${name}`], null);
+            const workflowsInfo = await services.info.getInfo();
+
+            setWorkflows(workflowList.items);
+            setColumns(workflowsInfo.columns);
+        };
+
+        getWorkflowsList();
```

Comment:
```suggestion
        (async () => {
            const workflowList = await services.workflows.list(namespace, null, [`${models.labels.cronWorkflow}=${name}`], null);
            const workflowsInfo = await services.info.getInfo();

            setWorkflows(workflowList.items);
            setColumns(workflowsInfo.columns);
        })();
```

I've just been using IIFEs in `useEffect` code like this (as it's shorter), so we can keep that consistent